src/gap_to_julia.jl

```diff
@@ -201,7 +200,13 @@ function gap_to_julia(
         new_array = Vector{T}(undef, len_list)
         recursion_dict[obj] = new_array
         for i = 1:len_list
-            current_obj = elmlist(obj, i)
+            if islist
+                current_obj = ElmList(obj, i)  # returns 'nothing' for holes in the list
+            else
+                # vector objects aren't lists,
+                # but the function for accessing entries is `ELM_LIST`
+                current_obj = ELM_LIST(obj, i)
+            end
```

Comment:
I wonder if we could achieve the same (but perhaps more elegantly?) by moving this loop into a separate helper function which takes `elmlist` as an argument -- then at least in theory, Julia should be able to generate optimal code in both cases, too?